explorer/templates.go

```diff
@@ -151,6 +151,42 @@ func float64Formatting(v float64, numPlaces int, useCommas bool, boldNumPlaces .
 	return []string{integer, dec[:places], dec[places:], trailingZeros}
 }
 
+func amountAsDecimalPartsTrimmed(v, numPlaces int64, useCommas bool) []string {
+	amt := strconv.FormatInt(v, 10)
+	intpart := v / 1e8
+	decpart := v % 1e8
+
+	if len(amt) <= 8 {
+		right := strings.TrimRight(amt, "0")
+		left := ""
+		tail := strings.Repeat("0", 8-len(amt))
+		if (8 - len(amt)) > 2 {
+			tail = "00"
+		}
+		return []string{"0", left + right, tail}
```

Comment:
why left + right if left is ""?  Why set left="" at all?

Reply:
Removed left